styles/core.scss

```
@@ -137,3 +137,139 @@ SPDX-License-Identifier: Apache-2.0 */
     }
   }
 }
+
+.core-page-title {
+  background: var(--background-default);
+  position: sticky;
+  top: 0;
+  padding-top: 0.5rem;
+  margin-top: 4rem;
+  margin-bottom: 0.25rem;
+  padding-bottom: 0;
+  width: 100%;
+  z-index: 5;
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: flex-end;
+  transition: padding 0.3s ease-out;
+
+  @media (max-width: v.$page-width-medium-threshold) {
+    top: 4rem;
+    padding-top: 0.75rem;
+    padding-bottom: 0.5rem;
```

What does this look like if we just use:

```suggestion
    padding: 0.75rem 0;
```